### PR TITLE
Filter events by date

### PR DESCRIPTION
**Pull request by ashkan18**
# Change 
We want to be able to compare today's current GMV with past days and weeks.

# Solution
Add option for querying events based on `start_date` , `end_date` and use it to show today's and yesterday's GMV in our live view dashboard.

# Selfie
localhost_4000_

### CODE REVIEW

**Comment by zephraph** [comment]
We should work on the grouping/styling of these a bit. It's weird that approved yesterday and today are on the top line to the right and the general yesterday and today gmv are widely spaced on the bottom. 

I also think we can remove the images and left align the columns of the approved/active orders

File: lib/apr_web/live/order_dashboard_live.ex [hunk 1]
```diff
@@ -19,28 +19,29 @@ defmodule AprWeb.OrderDashboardLive do
         <palette-jumbo label="Pending Approval Commission">
           <%= currency(@active_orders.totals.commission_cents) %>
         </palette-jumbo>
+
+        <palette-jumbo label="Approved Yesterday GMV">
```
[the remark this answers is not in the document]
Comment (zephraph) [reply lib/apr_web/live/order_dashboard_live.ex]: @SamRozen you can used GitHub's suggestion feature at the top of a comment bar to make it easy for others to accept. 

image

Like this 👇 

```suggestion
        <palette-jumbo label="Yesterday's GMV">
```

File: lib/apr_web/live/order_dashboard_live.ex [hunk 2]
```diff
@@ -19,28 +19,29 @@ defmodule AprWeb.OrderDashboardLive do
...
-            <img class="mb-1" src="<%= artwork["imageUrl"] %>" />
-            <div class="mb-0_5 sans-2-medium"> <%= currency(event.payload["properties"]["items_total_cents"]) %> </div>
-            <div class="serif-2-semibold color-black60"> <%= artwork["artist_names"] %> </div>
-            <div class="serif-2-italic color-black60"> <%= artwork["title"] %> </div>
-            <div class="serif-2 color-black60"> <%= artwork["partner"]["name"] %> </div>
-            <div class="serif-2 color-black30"> <a href="<%= exchange_link(event.payload["object"]["id"]) %>"> <%= event.payload["properties"]["code"] %></a></div>
-          </div>
-        <% end %>
-        </section>
+      <section class="main-stats">
```
[the remark this answers is not in the document]
Comment (ashkan18) [reply lib/apr_web/live/order_dashboard_live.ex]: yes! i copy pasted wrong thing :) removing these.

File: lib/apr_web/live/order_dashboard_live.ex (hunk 1 above)
[the remark this answers is not in the document]
Comment (ashkan18) [reply lib/apr_web/live/order_dashboard_live.ex]: i need to learn this magic suggestion thing too! my only issue with that is each change is one commit.

**Comment by ashkan18** [comment]
@zephraph going to merge this and lets work on it together to get it look better! i def need your 👀

**Pull request merged**